iOS/Podspecs/libevent.podspec

```diff
@@ -0,0 +1,565 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Because libevent is also relied on by Flipper and *with* OpenSSL bindings,
```

Comment:
So I take it this isn’t correct then? Seeing as I don’t see any changes to enable the libevent OpenSSL bindings, or did I overlook them?

Reply:
Yeah its nothing was required to be changed. I will remove that comment from the podspec file. Also you can now refer this pod from cocoapod direcctly. I have published it in the cocoapod repository.

Reply:
> I have published it in the cocoapod repository.

Ah perfect! :pray: